docker/Dockerfile

```diff
@@ -58,28 +36,37 @@ RUN conda update -qy conda \
       pip \
       python=3.7
 
-COPY . /app/alphafold
-RUN wget -q -P /app/alphafold/alphafold/common/ \
-  https://git.scicore.unibas.ch/schwede/openstructure/-/raw/7102c63615b64735c4941278d92b554ec94415f8/modules/mol/alg/src/stereo_chemical_props.txt
+COPY . /home/jupyter/alphafold
```

Comment:
Dont need to use a different location for the COPY; this was a red-herring I chased down.  Ill back these changes out.